source/Tests/Tests.csproj

```diff
@@ -13,7 +13,6 @@
 
     <PackageReference Include="Octopus.Data" Version="5.3.0" />
     <PackageReference Include="Octopus.Server.Extensibility" Version="12.1.0" />
-    <PackageReference Include="Octopus.TinyTypes" Version="0.1.313" />
```

Comment:
Is this deliberate?

Reply:
Yeah, got a dependency mismatch error - it's pulled in via `Octopus.Server.Extensibility` as well